frontend/packages/component-library/src/cms/image/Image.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import {ImgHTMLAttributes} from 'react';
+
+import moduleStyles from './image.module.scss';
+
+export interface ImageProps extends ImgHTMLAttributes<HTMLElement> {
+  /** Image source */
+  src: string;
+  /** Image alt text */
+  altText?: string;
+  /** Image border */
+  border?: boolean;
```

Comment:
nit: since it's boolean variable I think it's worth renaming this variable to `showBorder`, and `boxShadow` to `showBoxShadow` , `withBoxShadow`, `withBorder` or smth like that. Current naming feels like it expects border/boxShadow attribute value

Reply:
I was curious about this too. I think eventually we could support multiple color options for these which is why I have it as-is, but I'll change them since we can always update later if needed.